### DISCUSSION TITLE
graphicle - a visualisation for your cronicle events

### DISCUSSION THREAD

**Discussion by dheise-rheinenergie-trading-com** [jhuckaby/Cronicle#643] [Show and tell]
Since we now have far more than 1300 events set up in our cronicle and they make intensive use of the chaining function, we needed a lean solution to visualize these chains.
Therefore I wrote a small tool in Python that fetches all events via REST API and displays them in the browser. 
It is indispensable for us in our daily work - maybe also for some of you. 

The tool is available here: https://github.com/dheise-rheinenergie-trading-com/graphicle/tree/main

**Comment by jhuckaby** [comment]
Wow, this is amazing!  Great work!  👏🏻👏🏻👏🏻 

You might consider adding a `LICENSE.txt` file to your repo, so people know how it can be used.

**Reply by dheise-rheinenergie-trading-com** [reply]
Thanks for yout advice. License is added to the repo.
